packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -68,6 +68,10 @@ export type Capabilities = {|
 export default class Store extends EventEmitter<{|
   collapseNodesByDefault: [],
   componentFilters: [],
+  // TODO (inline errors) Do we even need the argument for subscriptions?
```

Comment:
I don't understand this TODO question.

Reply:
Right now we emit `store.errorsAndWarnings` but every event listener right now is ignoring that argument and using the store getter instead. In other words, we should probably remove the parameter since it's unused. People might get the false impression that the event payload contains a diff.